Tools/CISupport/ews-build/steps.py

```diff
@@ -2534,7 +2533,12 @@ def getQueueStatusFromList(self, change_id, pr_number):
             yield self._addToLog('stdio', f'Accessed {url} with unexpected status code {response.status_code}.\n')
             return defer.returnValue(False if response.status_code // 100 == 4 else None)
 
-        for queue in self.QUEUES_FOR_SAFE_MERGE_QUEUE:
```

Comment:
You should add a `#FIXME` comment here, because I think it's both of our assessment that you shouldn't need this, and skipping a queue should be a valid state for Safe Merge Queue to continue.